test/acceptance/user.acceptance.ts

```diff
@@ -0,0 +1,49 @@
+import { HelloWorldApp } from '../..';
+import { CoreBindings } from '@loopback/core';
+import { expect, supertest } from '@loopback/testlab';
+import { UserRepository } from '../../src/repositories/user.repository';
+import { User } from "../../src/models/user";
+
+describe('User (acceptance)', () => {
+  let app: HelloWorldApp;
+  let repository: UserRepository;
+  let request: supertest.SuperTest<supertest.Test>;
+
+  before(givenRunningApp);
+  before(givenEmptyDatabase);
+
+  it('retrieves user details', async () => {
+    // arrange
+    const user = await givenUser({
+      username: 'John'
+    });
+
+    // act
+    const response = await request.get('/users')
+
+    // assert
+    expect(response.body).to.deepEqual([{
+      username: 'John',
+      id: 1
+    }]);
+  });
+
+  async function givenEmptyDatabase() {
+    await repository.deleteAll();
+  }
+
+  async function givenRunningApp() {
+    app = new HelloWorldApp();
+    repository = await app.get('repositories.UserRepository');
+
+    app.bind(CoreBindings.HTTP_PORT).to(0);
```

Comment:
@kjdelisle have you released your REST refactor to npmjs yet? I think this will no longer work after your changes, will it?